tests/core/engine_adapter/test_integration.py

```diff
@@ -1898,3 +1898,85 @@ def test_dialects(ctx: TestContext):
     pd.testing.assert_frame_equal(
         df, pd.DataFrame([[1, 1, 1, 1]], columns=expected_columns), check_dtype=False
     )
+
+
+@pytest.mark.parametrize(
+    "time_column, time_column_type, time_column_format, result",
+    [
+        (
+            exp.null(),
+            exp.DataType.build("TIMESTAMP"),
+            None,
+            {
+                "default": None,
+                "bigquery": pd.NaT,
+                "databricks": pd.NaT,
+                "duckdb": pd.NaT,
+                "motherduck": pd.NaT,
+                "snowflake": pd.NaT,
+                "spark": pd.NaT,
+            },
+        ),
+        (
+            "2020-01-01 00:00:00+00:00",
+            exp.DataType.build("DATE"),
+            None,
+            {
+                "default": datetime(2020, 1, 1).date(),
+                "duckdb": pd.Timestamp("2020-01-01"),
+            },
+        ),
+        (
+            "2020-01-01 00:00:00+00:00",
+            exp.DataType.build("TIMESTAMPTZ"),
+            None,
+            {
+                "default": pd.Timestamp("2020-01-01 00:00:00+00:00"),
+                # TODO: Figure out why this is bytes and if this is really ok
```

Comment:
Unfortunately, this is an outstanding issue with pymssql https://github.com/pymssql/pymssql/issues/649

Reply:
Thanks for linking that! Will update the comment.